app/lib/features/desktop_setup/providers/desktop_setup_provider.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:launch_at_startup/launch_at_startup.dart';
+
+final desktopFeaturesProvider = StateProvider<bool>((ref) => false);
+
+final launchAtStartupProvider = Provider<LaunchAtStartup>(
+  (ref) => launchAtStartup,
+);
```

Comment:
I don't understand the need of to 2 providers here.

Instead of I would suggest one provider like this.

```suggestion
final launchAtStartupValueProvider = FutureProvider<bool>((ref) async {
  final launchAtStartup = LaunchAtStartup.instance;
  return await launchAtStartup.isEnabled();
});
```

Reply:
I used separate providers to keep things clear — one for the instance, one for the value.

I will change it to one